config/config-example.yaml

```diff
@@ -4,27 +4,236 @@ receivers:
     collection_interval: 1m
     scrapers:
       memory:
+      cpu:
+      disk:
+      filesystem:
+      network:
+      swap:
+      process:
+      load:
```

Comment:
For Linux there is also the `processes` scraper (this might need to be cleaned up a bit before its usable, but take a look if you have time)